wrappers/meta.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Wrappers",
+  "submodules": [
+    {
+      "name": "C Wrappers",
+      "elements": [
+        {
+          "name": "CCSDS Packet",
+          "additionalFiles": [
+            "asn1pusclib_globals.h",
+            "Crc.c",
+            "Crc.h",
```

Comment:
shouldn't path include `c/` ?

Reply:
btw - rebase to master, I had to change path to asn1scc complier on gdrive

Reply:
adding 'c/' would required change in source code, as currently it is assumed that file name is defined, and not part of the path.  Whole library directory is searched to find file with matching name, and than create path to it. It could be kept in mind, that if file names would not be unique, than behavior is undefined.

Reply:
ack